include/RAJA/util/Registry.hpp

```diff
@@ -91,14 +91,14 @@ namespace util {
 
     /// A static registration template.
     template <typename V>
-    class Add {
+    class add {
```

Comment:
What is the reason for the change to all lower case?  Typically, class names are capitalized.

Reply:
I can find a workaround for this. To change the name of the constructor from "Add" to "add" I also had to change the class name.

Reply:
OK. That's fine, then.